_sources/Conditionals/Logicaloperators.rst

```diff
@@ -61,11 +61,10 @@ is true if ``x > y`` is false, that is, if ``x`` is less than or equal to
    :answer_b: 0 &lt; x &lt; 5
    :answer_c: x &gt; 0 or x &lt; 5
    :answer_d: x &gt; 0 and x &lt; 5
-   :correct: d
+   :correct: b,d
    :feedback_a: Each comparison must be between exactly two values.  In this case the right-hand expression &lt; 5 lacks a value on its left.
-   :feedback_b: This is tricky.  Although most other programming languages do not allow this syntax, in Python, this syntax is allowed.  However, you should not use it.  Instead, make multiple comparisons by using and or or.
+   :feedback_b: Although most other programming languages do not allow this syntax, in Python, this syntax is allowed.  Even though it is possible to use this format, you should not use it all the time.  Instead, make multiple comparisons by using and or or.
    :feedback_c: Although this is legal Python syntax, the expression is incorrect.  It will evaluate to true for all numbers that are either greater than 0 or less than 5.  Because all numbers are either greater than 0 or less than 5, this expression will always be True.
    :feedback_d: Yes, with an ``and`` keyword both expressions must be true so the number must be greater than 0 an less than 5 for this expression to be true.
-   :practice: T
```

Comment:
The `:practice: T ` makes this question available to the practice tool.  Is there some reason for removing it?

Reply:
Sorry,

Completely forgot about that line. I removed it to see if it would change the formatting of the program since I was having some trouble at first getting multiple answers working. I can fix this soon.

Reply:
@RustyDotson -- just a reminder to put the practice flag back so I can accept this PR